tests/api/test_send_text_message.py

```diff
@@ -112,7 +112,7 @@ def test_broadcast_text_message(self):
         responses.add(
             responses.POST,
             LineBotApi.DEFAULT_API_ENDPOINT + '/v2/bot/message/broadcast',
-            json={}, status=200
+            json={}, status=200, headers={'X-Line-Request-Id': 'request_id_test'}
         )
 
         self.tested.broadcast(self.text_message)
```

Comment:
[ask]
is it unnecessary to check this response?

Reply:
Is it not enough to check at L148?
```python
self.assertEqual('request_id_test', response.request_id)
```